### ISSUE TITLE
Login Page Exposed to  SQL Injection Attack

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#54]
When I type like this,

![screen shot 2019-03-03 at 2 07 39 am](https://user-images.githubusercontent.com/25061080/53692189-41648080-3d59-11e9-874b-ccdbc8de3d35.png)

The SQL statement is executed like the following :
```
SELECT * FROM user_profile where email='mrliu@buffalo.edu' and password='';DROP TABLE user_profile;'
undefined
```

As the result, 

It could remove the data table

**Comment by JacobYim** [comment]
To prevent the SQL injection Attack, the following cases are blocked :
- email : the string having  ; , = , ; , or ' 
- password : the string having ; , =, ), ( , or  ' 
- Empty cases
- Blank cases

which is implemented at Merge Request #59

**Comment by JacobYim** [comment]
Test case for filter is needed. Login Filter test will be tested on the issue #60

**Comment by JacobYim** [comment]
This solution merged at #65.
